src/js/tabs/led_strip.js

```diff
@@ -567,7 +571,20 @@ led_strip.initialize = function (callback, scrollPosition) {
             }
 
             function save_to_eeprom() {
-                mspHelper.writeConfiguration(false);
+                mspHelper.writeConfiguration(false, save_completed);
+            }
+
+            function save_completed() {
+                const buttonDelay = 1500;
+
+                 // Allow firmware to make relevant changes before initialization
+                setTimeout(() => {
+                    saveButton.html(i18n.getMessage('buttonSaved'));
+
+                    setTimeout(() => {
+                        saveButton.html(oldText).removeClass('disabled');
+                    }, buttonDelay);
+                }, buttonDelay);
             }
```

Comment:
Repeated code if possible should probably go in /js/utils/common.js so we can easy extend to other tabs where applicable.

```comment
            function save_completed() {
                const buttonDelay = 1500;

                 // Allow firmware to make relevant changes before initialization
                setTimeout(() => {
                    saveButton.html(i18n.getMessage('buttonSaved'));

                    setTimeout(() => {
                        saveButton.html(oldText).removeClass('disabled');
                    }, buttonDelay);
                }, buttonDelay);
            }
```